<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmitterPluginPage</class>
 <widget class="QWidget" name="EmitterPluginPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_title">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Create Emitter Node</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="gbx_extraObjs">
       <property name="title">
        <string>Extra Objects</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="cbx_addObjs">
          <property name="text">
           <string>Add objects</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="gbx_emOptions">
       <property name="title">
        <string>Emitter Options</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="lbl_className">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Emitter Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_className">
          <property name="placeholderText">
           <string>Enter a class name for the Emitter</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QGroupBox" name="gbx_emitFn">
       <property name="title">
        <string>Emit Function</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>The Wizard will make an educated guess at a default function that could be useful for emitting particles</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbx_createEmit">
          <property name="text">
           <string>Create default Emit function</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
